addons/im_livechat/demo/im_livechat_channel/im_livechat_session_1.xml

```diff
@@ -95,5 +104,13 @@
         </record>
         <function model="discuss.channel" name="rating_apply"
             eval="([ref('im_livechat.livechat_channel_session_1')], 5, 'LIVECHAT_1', None, 'Good Job')"/>
+        <record id="livechat_channel_session_1_call_1" model="discuss.call.history">
```

Comment:
:pick: I'd rather having the call history created as close as possible from its notification.